styles/amplenote/catppuccin.user.css

```
@@ -0,0 +1,190 @@
+/* ==UserStyle==
+@name Amplenote Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/amplenote
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/amplenote
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/amplenote/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aamplenote
+@description Soothing pastel theme for Amplenote
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("amplenote.com") {
+  :root:not(:has(body.theme-light)) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root:has(body.theme-light) {
```

```suggestion
  body:not(.theme-light) {
    #catppuccin(@darkFlavor, @accentColor);
  }
  body.theme-light {
```
Is there a reason these can't be simplified as such? We should ideally be able to avoid `:has()` here. I believe this change will require you to change `body.themeable {` below to `&.themeable {`.